presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -166,11 +166,16 @@
     void beginQuery(Session session, Collection<TableHandle> tableHandles);
 
     /**
-     * Notify catalogName of the execution of a query
+     * Start a query
```

Comment:
Please have a look on https://github.com/prestodb/presto/pull/6114/ since I rebased this branch on the most recent master so the diff in GH on this PR is not reliable anymore.